### ISSUE TITLE
Segmentation fault when using fmeasure as metric

### ISSUE THREAD

**Issue by jnphilipp** [fchollet/keras#5156]
Hi, I'm getting a segmentation fault when I add fmeasure to the metric list.

```
inputs = Input(shape=(30, 20))
x = Flatten()(inputs)
predictions = Dense(20, activation='softmax')(x)

model = Model(input=inputs, output=predictions)
model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy', 'fmeasure'])

X = np.random.rand(100, 30, 20)
y = np.eye(100, 20)

history = model.fit(X, y)

```

**Comment by davidslater** [comment]
I have also recently experienced this problem.

**Comment by yhenon** [comment]
Can you share your version/backend? Works fine for me. Output here:


	Using Theano backend.
	Using gpu device 0: GeForce GTX TITAN X (CNMeM is disabled, cuDNN 5005)
	Epoch 1/10
	100/100 [==============================] - 0s - loss: 0.6555 - acc: 0.0900 - fmeasure: 0.0000e+00     
	Epoch 2/10
	100/100 [==============================] - 0s - loss: 0.5619 - acc: 0.1100 - fmeasure: 0.0000e+00     
	Epoch 3/10
	100/100 [==============================] - 0s - loss: 0.5104 - acc: 0.0800 - fmeasure: 0.0000e+00     
	Epoch 4/10
	100/100 [==============================] - 0s - loss: 0.4653 - acc: 0.0600 - fmeasure: 0.0000e+00     
	Epoch 5/10
	100/100 [==============================] - 0s - loss: 0.4265 - acc: 0.0900 - fmeasure: 0.0000e+00     
	Epoch 6/10
	100/100 [==============================] - 0s - loss: 0.3827 - acc: 0.1400 - fmeasure: 0.0000e+00     
	Epoch 7/10
	100/100 [==============================] - 0s - loss: 0.3514 - acc: 0.1800 - fmeasure: 0.0000e+00     
	Epoch 8/10
	100/100 [==============================] - 0s - loss: 0.3161 - acc: 0.1800 - fmeasure: 0.0000e+00     
	Epoch 9/10
	100/100 [==============================] - 0s - loss: 0.2816 - acc: 0.1900 - fmeasure: 0.0800         
	Epoch 10/10
	100/100 [==============================] - 0s - loss: 0.2529 - acc: 0.2000 - fmeasure: 0.1067

**Comment by davidslater** [comment]
keras verstion 1.2.0
theano version 0.8.2
cuDNN 5105

**Comment by nouiz** [comment]
Use Theano dev version.

**Comment by jnphilipp** [comment]
I used Theano 0.9.0dev5 with a libgpuarray 0.6rc.

**Comment by nouiz** [comment]
Can you start Python in HDB and print the stack trace? Something like

gdb Python
>  r yourscript.py
This will tell you which software is the cause.

If it is a file in Theano, join it with the stack trace.
